File: Java/demo/src/main/java/com/exercicios/Exercicio20.java
//Algoritmo que gera e escreve números ímpares dos números entre 100 e 200.
package com.exercicios;

public class Exercicio20 {
    public static void main(String[] args) {
        //Variável
        int i;

        //Processamento
        for (i = 100; i <= 200; i++) {
            if (i % 2 != 0) {
                System.out.println("Ímpar: " + i);
            }
        }
    }
}

File: C/Exercicio20.c
//Algoritmo que gera e escreve os números ímpares dos números 100 e 200.
#include <stdio.h>

int main() {
    //Variável
    int i;

    //Processamento
    for (i = 100; i <=200; i++) {
        if (i % 2 != 0) {
            printf("Número ímpar: %d \n", i);
        }
    }
    return 0;
}